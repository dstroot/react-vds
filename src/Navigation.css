.navbar {
  box-shadow: 0 2px 40px 0 rgba(0,0,0,0.08);
  background-color: rgba(255,255,255,.98);
  -webkit-backdrop-filter: blur(20px);
  font-weight: 300;
  font-size: .875rem;
  text-transform: uppercase;
}

/* Show it is fixed to the top */
body {
  min-height: 75rem;
  padding-top: 5rem;
}