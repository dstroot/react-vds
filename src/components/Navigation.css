.navbar {
  box-shadow: 0 2px 40px 0 rgba(0, 0, 0, 0.08);
  background-color: rgba(255, 255, 255, 0.98);
  -webkit-backdrop-filter: blur(20px);
  font-weight: 300;
  font-size: 0.875rem;
  text-transform: uppercase;
}

/* Show it is fixed to the top */
body {
  min-height: 75rem;
  padding-top: 56px;
}
