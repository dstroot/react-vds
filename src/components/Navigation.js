import React from 'react';
import { NavLink } from 'react-router-dom';

import logo from '../media/logo.png';
import './Navigation.css';

const Navigation = () => {
  return (
    <header className="navbar navbar-expand navbar-light flex-column flex-md-row fixed-top">
      <NavLink to="/" className="navbar-brand mr-0 mr-md-2">
        <img
          className="d-inline-block align-top"
          src={logo}
          width="30"
          height="30"
          alt=""
        />{' '}
        Verite Data Science
      </NavLink>
    </header>
  );
};

export default Navigation;
