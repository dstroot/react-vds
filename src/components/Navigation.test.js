import React from 'react';
import Navigation from './Navigation';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';

test('Navigation renders a Navbar', () => {
  const component = renderer.create(
    <Router>
      <Navigation />
    </Router>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
