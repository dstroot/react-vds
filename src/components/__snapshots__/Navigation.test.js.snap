// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation renders a Navbar 1`] = `
<nav
  className="navbar fixed-top navbar-expand-sm navbar-light bg-light"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="/"
      onClick={[Function]}
    >
      <img
        alt=""
        className="d-inline-block align-top"
        height="30"
        src="logo.png"
        width="30"
      />
       
      Verite Data Science
    </a>
    <button
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="navbar-collapse collapse"
    >
      <a
        aria-current={null}
        className="nav-link"
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
      <a
        aria-current={null}
        className="nav-link"
        href="/contact"
        onClick={[Function]}
      >
        Contact
      </a>
    </div>
  </div>
</nav>
`;
